packages/klevu-ui/src/components/klevu-product/klevu-product.tsx

```diff
@@ -288,11 +300,29 @@ export class KlevuProduct {
             </slot>
           </a>
           <slot name="bottom"></slot>
+          {this.variant === "line" && <slot name="ratings">{this.#renderRatings()}</slot>}
         </div>
       </Host>
     )
   }
 
+  #renderRatings() {
+    return (
+      this.showRatings && (
+        <div class="ratings">
```

Comment:
should the `<slot name="ratings">` be moved here? So render code would look like this: `{this.variant === "line" && this.#renderRatings()}` 

And if moved here then we do not render slot when ratings is not enabled.

Reply:
I wanted to give user ability to decide if they want to show their own component in this slot, then they are responsible for showing hiding the ratings. 
If we render it then we use our logic.

Reply:
True. So let's leave this as is.